fe/fe-core/src/main/java/com/starrocks/catalog/Catalog.java

```diff
@@ -4497,8 +4497,24 @@ public static void getDdlStmt(String dbName, Table table, List<String> createTab
         // 1.1 view
         if (table.getType() == TableType.VIEW) {
             View view = (View) table;
-            sb.append("CREATE VIEW `").append(table.getName()).append("` AS ").append(view.getInlineViewDef());
-            sb.append(";");
+            sb.append("CREATE VIEW `").append(table.getName()).append("`\n(\n");
```

Comment:
In the show statement, it is all explicit on one line, and it is not recommended to add a “\n"